src/pages/actions/[slug].tsx

```diff
@@ -269,6 +272,7 @@ export default function ActionPage() {
         </Container>
       </HeaderSection>
       <Container fluid="lg">
+        {refetching && <Loader />}
         <ActionPlotCard>{actionPlot}</ActionPlotCard>
```

Comment:
The `Loader` component isn't being contained by the `Container` here so it covers the entire page and main navigation. This looks like the correct place for it, so that probably means the `Container` doesn't have a set position – try adding `position: relative;` to the container.

<img width="1263" alt="Screenshot 2023-11-06 at 17 59 01" src="https://github.com/kausaltech/kausal-paths-ui/assets/15343658/cf873911-91d3-49d3-95f6-d94653a3b107">